src/test/java/io/hexlet/typoreporter/service/TypoServiceIT.java

```diff
@@ -165,28 +165,28 @@ void deleteTypoByIdNotFound(final Long id) {
     @Test
     void getCountTypoByStatusForWorkspaceName() {
         List<Pair<TypoStatus, Long>> listWks = service.getCountTypoByStatusForWorkspaceName(WORKSPACE_101_NAME);
-        assertThat((long) listWks.size()).isEqualTo(4L);
+        assertThat(listWks.size()).isEqualTo(4);
         assertThat(listWks.stream()
```

Comment:
You can use `filteredOn` or `extracting` methods from AssertJ instead of `java.Stream`

Reply:
Example: [AssertJ List](https://reflectoring.io/assertj-lists/)